src/main/java/core/basesyntax/Buldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Buldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Machine - buldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Machine - buldozer stopped working");
```

Comment:
Similarly to the `doWork` method, the message in `stopWork` should also be more informative. It should clearly indicate that it is the `Bulldozer` that has stopped working, for example: 'Bulldozer has stopped working.'